notNeededPackages.json

```
@@ -3766,6 +3766,10 @@
             "libraryName": "rollup-plugin-node-resolve",
             "asOfVersion": "4.1.0"
         },
+        "rollup-plugin-postcss": {
+            "libraryName": "rollup-plugin-postcss",
+            "asOfVersion": "2.5.0"
```

```suggestion
            "asOfVersion": "3.1.4"
```

Types were included in the main package as of version 3.1.4
See https://unpkg.com/browse/rollup-plugin-postcss@3.1.3/ vs https://unpkg.com/browse/rollup-plugin-postcss@3.1.4/

Also taken from the DT README:
> * `<asOfVersion>`: A stub will be published to `@types/<typingsPackageName>` with this version. Should be higher than any currently published version, and should be a version of `<libraryName>` on npm.

---

Good catch, you're absolutely right.